src/include/homestore/replication/repl_dev.h

```diff
@@ -61,51 +69,89 @@ struct repl_req_ctx : public boost::intrusive_ref_counter< repl_req_ctx, boost::
     friend class SoloReplDev;
```

Comment:
Can we also put a important comment on top of `repl_req_ctx` that we should avoid putting members in public area because we are trying to achieve safe access for this structure so that engineers from other area can also achieve this goal in the future.

Reply:
Done.